noggin/tests/unit/conftest.py

```diff
@@ -35,6 +35,15 @@ def ipa_cert():
         yield
 
 
+@pytest.fixture(scope="session")
+def add_hidden_group():
+    hidden_group = app.config.get('HIDDEN_GROUPS')
+    parent_group = ipa_admin.group_find(hidden_group)
+    if not parent_group:
+        ipa_admin.group_add(cn=hidden_group)
```

Comment:
Please also set the `nonposix` keyword argument to `True`, we don't need that group to be posix.